src/js/components/covid19/assistanceListing/SpendingByCFDA.jsx

```diff
@@ -61,6 +61,13 @@ const SpendingByCFDA = () => {
         Analytics.event({ category: 'COVID-19 - Award Spending by CFDA', action: `${activeTab} - click` });
     };
 
+    // Keep track of previous tab to prevent duplicate requests in summary
+    const prevTabRef = useRef();
+    useEffect(() => {
+        prevTabRef.current = activeTab;
+    });
+    const prevTab = prevTabRef.current;
```

Comment:
In this case, we're setting the `prevTab` to equal the value of a ref, which itself is the value of a property in state called `activeTab`. So `prevTab` equals `activeTab`? This is self-refuting and confusing. Not sure what exactly we're trying to do here, but I think we're `trying` to capture stale state? This shouldn't be necessary.

Reply:
https://reactjs.org/docs/hooks-faq.html#how-to-get-the-previous-props-or-state